webpack-test/configCases/split-chunks/chunk-filename-delimiter-default/test.filter.js

```diff
@@ -1 +1 @@
-module.exports = () => {return false}
\ No newline at end of file
+module.exports = () => 'splitted chunk ids should be the same as webpack'
```

Comment:
better link an issue with priority, and add to #4327

Reply:
It's already there https://github.com/web-infra-dev/rspack/issues/4334

Reply:
I mean replace this with the issue's link, instead with the plain text, https://github.com/web-infra-dev/rspack/pull/4337#issuecomment-1764688831 so this can be list in the diff comment

Reply:
Done